CONTRIBUTING.md

```diff
@@ -109,16 +109,90 @@ describe("TESTS: Path => To => Tested => Resource", () => {
     cy.verifySuccess();
     // Checks that /path=to/tested=resource:read-attribute(name=module)
     // returns "sample" and is succesful
-    cy.verifyAttribute(
-      managementEndpoint,
-      ["path", "to", "tested", "resource"],
-      "module",
-      "sample"
-    );
+    cy.verifyAttribute(managementEndpoint, ["path", "to", "tested", "resource"], "module", "sample");
   });
 });
 ```
 
+### All custom method
```

Comment:
I think this section (and all other sections) could be removed since IMHO they should be part of https://github.com/hal/berg/issues/25. WDYT?

Reply:
Yes, There are duplicate information mentioned also in readme.  Removed this and following sections.